Menu/README.md

```diff
@@ -0,0 +1,69 @@
+
+# Menu Interface
+
+**Status**: Proposed
+
+| Name      | ID                                                                |
+| :-------- | :---------------------------------------------------------------- |
+| Menu      | ac1a4d3ecea232e49783df4a23a81823cdca3205dc58cd20c4db259c25605b48  |
+
+
+## Description
+
+Interface for menu.
+
+## Functions
+
+`select` - prints prompt message to the user and returns string entered by user
+
+arguments: 
+
+	title: bytes
+	
+	description: bytes
+
+    items: MenuItem[]
+
+returns: 
+
+	void
+
+
+## Declaration in Solidity
+
+```jsx
+
+struct MenuItem {
+        string title;
+        string description;
+		uint32 handlerId;
+    }
+
+interface IMenu {
+    
+	function select(string title, string description, MenuItem[] items) external returns (uint32 index);
+}
+```
+
+## Declaration in C++
+
+```cpp
+namespace tvm { namespace schema {
+
+__interface IMenu {
+
+	[[internal]]
+	void select();
```

Comment:
For c++ it is really better to have return value and printf-like arguments:

```
auto sel = cout().select({
  { "Run something for address {address}", {addr}, "Run something 1", {}, 1 },
  { "Run something for value {value}", {val}, "Run something 2", {}, 2 }
});
switch (sel) {
case 1:
  run_something1();
  break;
case 2:
  run_something2();
  break;
}
```

Reply:
Now `select` returns index of selected menu item. I think we don't need to complicate title and description with a printf-like syntax. Can you use something similar to solidity's `format` function in c++? Or we can use Terminal.printf() if needed.